build-support/bin/release.sh

```diff
@@ -572,36 +583,51 @@ function build_pex() {
   local linux_platform_noabi="linux_x86_64"
   local osx_platform_noabi="macosx_10.11_x86_64"
 
+  dest_suffix="py27.pex"
+  if [[ "${python_three:-false}" == "true" ]]; then
+    dest_suffix="py36.pex"
+  fi
+
   case "${mode}" in
     build)
       case "$(uname)" in
         # NB: When building locally, we use a platform that does not refer to the ABI version, to
         # avoid needing to introspect the python ABI for this machine.
         Darwin)
-          local platforms=("${osx_platform_noabi}")
+          local platform=("${osx_platform_noabi}")
           ;;
         Linux)
-          local platforms=("${linux_platform_noabi}")
+          local platform=("${linux_platform_noabi}")
```

Comment:
These were bad renames from #7393. Renaming it to `platforms` meant the `dest` and `stable_dest` had an undefined variable.